setup.py

```diff
@@ -36,4 +36,19 @@
               ['schema/' + x for x in os.listdir('schema')]),
       ],
       test_suite='snapcraft.tests',
+      license='GPL v3',
+      classifiers=(
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Natural Language :: English',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
```

Comment:
That's a miss, I was waiting for rmadison to return and forgot to update, it should be just 3.4:

    python3 | 3.4.0-0ubuntu2   | trusty          | amd64, arm64, armhf, i386, powerpc, ppc64el
    python3 | 3.4.3-1          | vivid           | amd64, arm64, armhf, i386, powerpc, ppc64el
    python3 | 3.4.3-4ubuntu1   | wily            | amd64, arm64, armhf, i386, powerpc, ppc64el